contrib/docker/build.sh

```diff
@@ -12,4 +12,5 @@ IMAGE_TAG="$1"
 here=$(dirname $(realpath "$0" 2> /dev/null || grealpath "$0"))
 . "$here"/../base.sh
 
-docker build --build-arg MAKEFLAGS="-j $WORKER_COUNT" -t "$IMAGE_TAG" -f contrib/docker/Dockerfile "$here"/../..
+docker buildx build --build-arg MAKEFLAGS="-j $WORKER_COUNT" -t "$IMAGE_TAG" -f contrib/docker/Dockerfile \
+    --platform linux/arm64/v8,linux/amd64 --push "$here"/../..
```

Comment:
```suggestion
docker buildx build --build-arg MAKEFLAGS="-j $WORKER_COUNT" -t "$IMAGE_TAG" -f contrib/docker/Dockerfile \
    --platform linux/arm64/v8,linux/amd64 "$here"/../..
```

Reply:
I've tried this, the images are then getting stuck in the buildx cache and not loaded into `docker image ls` unfortunatelly. The option `--load` shoud do exactly that, but it does not support the multiarch builds :/

Reply:
Oh ok. So... what do I do here? Just ignore the error I got? I guess to fix the error I got I need to do something something authenticate to dockerhub something?

I am not even sure who or what calls this script, LOL.

All I care about with this particular `Dockerfile` is that travis works, which it seems to now so.  We can nix this suggestion.

Sorry for nub questions and your help is appreciated -- so you are telling me it's ok to leave the script in a 'not working by default unless you log in' state like this?  If you say so, we'll do that.

Reply:
I guess I can pimp up the script to output a message about this condition explicitly, also add a readme if needed.